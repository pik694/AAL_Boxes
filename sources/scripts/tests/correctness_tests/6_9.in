6
81 46 90
71 95 94
41 20 61
12 68 97
77 12 38
73 33 32

