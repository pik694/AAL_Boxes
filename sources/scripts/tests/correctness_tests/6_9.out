4 2 5 0 1 3 
1210742->798314
